src/edu/washington/escience/myria/daemon/MyriaDriverLauncher.java

```diff
@@ -212,9 +214,60 @@ private static int getDriverMemory(final Configuration conf) throws InjectionExc
   }
 
   /**
+   * Prints the expected configuration, compared to the actual configuration given.
+   * Used for logging command line arguments.
+   */
+  private static void logStartupMessage(Class<? extends Name<?>>[] classes, Injector inj) {
```

Comment:
I'd prefer a method that didn't mix computation and side effects. How about this method just generates the log message and the caller logs it?